Script/AtomicNET/AtomicNET/Atomic2D/PhysicsWorld2D.cs

```diff
@@ -0,0 +1,27 @@
+﻿using System;
+using System.Runtime.InteropServices;
+
+namespace AtomicEngine
+{
+    [StructLayout(LayoutKind.Sequential)]
+    public struct PhysicsRaycastResult2D
+    {
```

Comment:
Good stuff, and consistent with 2D/3D physics C++ subsystems, which counts for something.  If we're incongruent with Box2D in places, that is ok, we don't expose the API, at least not in scripting.  It is fully available in C++ for customization, etc.  I think your Box2D experience is valuable here, I haven't done a ton with it outside of examples. :+1: